### DISCUSSION TITLE
Small bbox 시각화 EDA

### DISCUSSION THREAD

**Discussion by JiyouSeo** [boostcampaitech2/object-detection-level2-cv-06#40] [Show and tell]
Small bbox들만 골라서 시각화한 내용을 EDA에 추가하였습니다.
Link: [code][codelink]

[codelink]:https://github.com/boostcampaitech2/object-detection-level2-cv-06/blob/master/EDA/eda_john.ipynb
anno converter로 작은 크기의 bbox를 제거하기 전에 간단하게 시각화해보시면서 어느 크기까지 제거할지 감을 잡아보아도 괜찮을 것 같습니다.
w_thres, h_thres를 조절하시면 원하는 max 크기의 small bbox를 추려낼 수 있습니다.

**Comment by ppskj178** [comment]
이런거 편하게 하려고 viewer를 만든건데... 
조건 달라질때마다 코드를 새로짜는거보다
범용성있는 뷰어를 완성시키는게 좋을것 같습니다.

전에 말했듯이 오브젝트단위 출력은 판다스문법으로 편하게 사용가능한데 이미지단위 출력은 조금 수정이 필요한 상태입니다.

혹시 이런방식은 별로신걸까요?

**Reply by ppskj178** [reply]
그리고 이제부터는 연습도할겸 코드 공유도할겸

코드 배포시에는 풀리퀘스트로 하면 좋을 것 같습니다.

**Reply by JiyouSeo** [reply]
아.. 알겠습니다.. 그럼 이미지 단위 출력에 대해서 제가 작업해볼까요?
네, 풀리퀘스트로 진행하도록 하겠습니다.

**Reply by ppskj178** [reply]
> 아.. 알겠습니다.. 그럼 이미지 단위 출력에 대해서 제가 작업해볼까요? 네, 풀리퀘스트로 진행하도록 하겠습니다.

해주시면 환영입니다!
아마 안해주시면 제가 필요해질때 하지않을까 싶어요.... ㅋㅋ

현재 이미지 단위 출력은 인덱스를 직접 할당해야 되서

굳이 하려면 아래 bbox 단위 출력에 있는 df를 잘 만져서 인덱스만 추려서 쓰면되긴합니다...
현재 제가 그러고있구요.. 근데 범용적이진않아서...

수정하게되면 df를 맨 위로 올리고 다 맨 아래처럼 쿼리만 할당해주면 알아서 출력되도록 해줄 생각이였습니다...

구현하게 되시면 참고 부탁드려요 

p.s 필요한 판다스 테크닉은 제 EDA/viewer/anno_converter 들에 거의 있을거라 참고하시면 좋을듯 합니다
